src/SprykerEco/Yves/Unzer/Theme/default/components/molecules/unzer-credit-card/unzer-credit-card.ts

```diff
@@ -0,0 +1,130 @@
+declare var unzer: unzerInterface;
+
+interface unzerInterface {
+    (publicKey: string, options: object): void;
+}
+
+interface cardInterface {
+    create: {
+        (inputField: string, {
+            containerId: string,
+            onlyIframe: boolean
+        }): void;
+    };
+    createResource: any;
+}
+
+import Component from 'ShopUi/models/component';
+import ScriptLoader from 'ShopUi/components/molecules/script-loader/script-loader';
+
+const CURRENT_PAYMENT_METHOD = 'unzerCreditCard';
+
+export default class UnzerCreditCard extends Component {
+    scriptLoader: ScriptLoader;
+    form: HTMLFormElement;
+    publicKeyInput: HTMLInputElement;
+    publicKey: string;
+    transactionIdInput: HTMLInputElement;
+    errorElement: HTMLElement;
+    card: cardInterface;
+
+    protected readyCallback(): void {}
+
+    protected init(): void {
+        this.scriptLoader = <ScriptLoader>Array.from(document.getElementsByClassName(`${this.jsName}__script-loader`))[0];
```

Comment:
Why is the `document` used instead of `this`?
```suggestion
        this.scriptLoader = <ScriptLoader>Array.from(this.getElementsByClassName(`${this.jsName}__script-loader`))[0];
```